src/providers/node/mod.rs

```diff
@@ -151,32 +169,56 @@ impl NodeProvider {
     }
 
     pub fn get_start_cmd(app: &App, env: &Environment) -> Result<Option<String>> {
+        let dlx = NodeProvider::get_package_manager_dlx_command(app);
+        let pkg_manager = NodeProvider::get_package_manager(app);
+        let executor = if pkg_manager == *"bun" { "bun" } else { "node" };
+        let package_json: PackageJson = app.read_json("package.json").unwrap_or_default();
+
         if Nx::is_nx_monorepo(app, env) {
             if let Some(nx_start_cmd) = Nx::get_nx_start_cmd(app, env)? {
                 return Ok(Some(nx_start_cmd));
             }
         }
+        if Turborepo::is_turborepo(app) {
```

Comment:
Can this block (and the build command block above), be moved to the `turborepo.rs` file (similar to how Nx works). 

```rs
if Turborepo::is_turborepo(app) {
  if let Some(turbo_start_cmd) = Turborepo::get_turbo_start_cmd(app, env, pkg_manager)? {
    return Ok(Some(turbo_start_cmd))
  }
}
```

It would also allow us to easily fallback to the regular node provider behaviour if no Turbo specific start/build commands can be found.

Reply:
Much cleaner! 🎉